Cargo.lock

```diff
@@ -1929,10 +1937,10 @@ name = "snarkvm-circuits"
 version = "0.7.5"
 dependencies = [
  "anyhow",
- "blake2",
+ "blake2 0.9.2",
```

Comment:
`snarkvm-circuits` and `snarkvm-polycommit` seem to still be using the older version of `blake2`

Reply:
Good catch, it isn't being used in code so it can likely be removed from the deps 👍